app/assets/javascripts/application.js.coffee

```diff
@@ -197,3 +197,9 @@ $ ->
     new ConfirmDangerModal(form, text)
 
   new Aside()
+
+  $('.hide-project-limit-message').on 'click', (e) ->
+    path = '/'
+    $.cookie('hide_project_limit_message', 'false', { path: path })
+    $(@).parents('.project-limit-message').remove()
+    e.preventDefault()
```

Comment:
I was put this in a partial, just like `_no_ssh.html.haml`, and render it anywhere the "New Project" button shows up.